AutoGraph/WebSocketClient.swift

```diff
@@ -187,6 +193,7 @@ open class WebSocketClient {
             self.attemptReconnectCount -= 1
         }
         self.reconnecting = true
+        self.state = .reconnecting
```

Comment:
why was this added? should we remove `self.reconnecting = true` then?

Reply:
So we can present to the user when we are reconnecting